zebra/zebra_rib.c

```diff
@@ -4216,6 +4216,30 @@ static int handle_pw_result(struct zebra_dplane_ctx *ctx)
 	return 0;
 }
 
+/*
+ * Handler for traffic control update results
+ */
+static int zebra_tc_dplane_result(struct zebra_dplane_ctx *ctx)
```

Comment:
I would remove this handler for now and just put the `dplane_ctx_fini()` in the calling function of this

We will add this handler back to another file when we have code to maintain internal state there